Makefile

```diff
@@ -176,36 +156,36 @@ JSON_YML_VALIDATE = poetry run python src/meltano/core/utils/validate_json_schem
 
 args = `arg="$(filter-out $@,$(MAKECMDGOALS))" && echo $${arg:-${1}}`
 
-lint_python:
+lint_python: ## Run Python linters & automatically apply fixes where possible
 	${JSON_YML_VALIDATE}
 	${TOX_RUN} fix -- $(call args)
 
-lint_eslint: ${MELTANO_WEBAPP}/node_modules
+lint_eslint: ${MELTANO_WEBAPP}/node_modules ## Run eslint & automatically apply fixes where possible
 	${ESLINT_RUN} --fix
 
-show_lint_python:
+show_lint_python: ## Run Python linters & display output
 	${TOX_RUN} lint -- $(call args)
 
-show_lint_eslint: ${MELTANO_WEBAPP}/node_modules
+show_lint_eslint: ${MELTANO_WEBAPP}/node_modules ## Run eslint & display output
 	${ESLINT_RUN}
 
-lint: lint_python lint_eslint
-show_lint: show_lint_python show_lint_eslint
+lint: lint_python lint_eslint ## Run linters & automatically apply fixes where possible
+show_lint: show_lint_python show_lint_eslint ## Run linters & display output
 
 # Makefile Related Tasks
 # ======================
 #
-# - `make explain_makefile` will bring up a web server with this makefile annotated.
-explain_makefile:
-	docker stop explain_makefile || echo 'booting server'
-	${DOCKER_RUN} --name explain_makefile -p 8081:8081 node ./Makefile_explain.sh
+# - `make explain` will bring up a web server with this makefile annotated.
+explain: ## Bring up a web server with this Makefile annotated
+	docker stop explain || echo 'booting server'
```

Comment:
IMHO we could nuke `make explain` and the associated bash script as part of this PR.